src/js/tracks/text-track-display.js

```diff
@@ -319,6 +320,23 @@ class TextTrackDisplay extends Component {
       }
       this.updateForTrack(descriptionsTrack);
     }
+
+    if (browser.IS_SMART_TV && !window.CSS.supports('inset', '10px')) {
```

Comment:
Should this be any browser doesn't support `inset`, and not just smart TVs? Smart TVs are the only real reason to be using such old browser versions, but it's not inconceivable that people might test things destined for a smart TV on an old browser on desktop/browserstack etc.

Reply:
Yeah, should be a feature test only, unless there's a good reason to only target TVs?
Below should be updated to match as well

Reply:
Updated the PR to support this changes on any browser, I got to move the update of the styles of textTrackDisplay to add support when user switches to full screen, can this be re-reviewed?